### ISSUE TITLE
Support for 3 level navigation

### ISSUE THREAD

**Issue by Jimmi08** [BlackrockDigital/startbootstrap-modern-business#58]
Could you add example how to solve 3 level navigation? 
I couldn't find any example when using divs for second level.  
Thank you

**Comment by zeufparis** [comment]
Hi,
3 level in the navbar ?

**Comment by Jimmi08** [comment]
> Hi,
> 3 level in the navbar ?

Yes, but without the next javascript,  existing solutions use ul/li tree, but this template use div on the second level.  I really can't remember now, but we ported this template for e107 CMS (navigation is generated) and I couldn't get markup for the 3rd level correctly. It's not supported by bootstrap 4, it was easy in BT3 to add, but users ask for this.
